<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite" duration-ms="64133" started-at="2019-06-16T13:58:06Z" finished-at="2019-06-16T13:59:10Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="64133" started-at="2019-06-16T13:58:06Z" finished-at="2019-06-16T13:59:10Z">
      <class name="com.zoho.testcases.BaseTest">
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.zoho.testcases.BaseTest@506e6d5e]" name="teardown" is-config="true" duration-ms="0" started-at="2019-06-16T19:29:10Z" finished-at="2019-06-16T19:29:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.zoho.testcases.BaseTest -->
      <class name="com.zoho.testcases.LoginTest">
        <test-method status="PASS" signature="HomeLoginTest(java.util.HashMap)[pri:0, instance:com.zoho.testcases.LoginTest@2d6e8792]" name="HomeLoginTest" duration-ms="11404" started-at="2019-06-16T19:28:07Z" data-provider="dp" finished-at="2019-06-16T19:28:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=nehadahiya88@yahoo.in, Password=Neha888#}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomeLoginTest -->
        <test-method status="FAIL" signature="HomeLoginTest(java.util.HashMap)[pri:0, instance:com.zoho.testcases.LoginTest@2d6e8792]" name="HomeLoginTest" duration-ms="51344" started-at="2019-06-16T19:28:18Z" data-provider="dp" finished-at="2019-06-16T19:29:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Username=nehadahiya88@yahoo.in, Password=Neha88#}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login unsuccessful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login unsuccessful
	at org.testng.Assert.fail(Assert.java:93)
	at com.zoho.Pages.WelcomePage.isLoginSuccessfull(WelcomePage.java:16)
	at com.zoho.testcases.LoginTest.HomeLoginTest(LoginTest.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- HomeLoginTest -->
      </class> <!-- com.zoho.testcases.LoginTest -->
    </test> <!-- Command line test -->
  </suite> <!-- Command line suite -->
</testng-results>
